README.md

```diff
@@ -148,14 +148,25 @@ root: ~/
 # Controls whether the tmux session should be attached to automatically. Defaults to true.
 # attach: false
 
+# Enables the display of pane titles. For example "editor" below. Defaults to false.
+# enable_pane_titles: true
+
+# Configures pane title position. Can be: bottom, top, or off. Defaults to top.
+# pane_title_position: bottom
```

Comment:
I tried setting this to `pane_title_position: off` while keeping `enable_pane_titles: true`, and I still see pane titles. I wonder if you could try it and confirm the behavior?

Reply:
Oh ya, it's because `off` is actually a yaml keyword in [yaml 1.1](https://yaml.org/type/bool.html) which is interpreted as a boolean false. If you want to pass the off string, you have to explicity quote it `"off"`.

I'm not sure how we should address this here. We could just say we don't officially support off, since it's not very useful, and will lead to confusion.

Reply:
Or we could re-interpret a boolean false as an "off" string.

Reply:
Makes sense! I think it's probably fine as is - just a small yaml gotcha.

Another case I thought of is if the user has pane titles enabled in their `.tmux.conf`, but `enable_pane_titles: false` in the tmuxinator project yaml, the pane titles still show in that session as it's currently written. That said, since the default is `false`, it wouldn't be a great user experience for pane titles to be turned off if you're expecting them on as defined in your `.tmux.conf`.

Reply:
Ya I think it's fine for .tmux.conf to take precedence.

Reply:
I will add a comment about the `off` thing just to try and reduce confusion.